torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -391,6 +392,54 @@ void mm_get_thread_blocking(
   return;
 }
 
+void mm_get_cache_blocking(
+    int num_threads,
+    int64_t M,
+    int64_t N,
+    int64_t K,
+    int64_t M0,
+    int64_t N0,
+    int64_t K0,
+    int64_t Mt_blocks,
+    int64_t Kt_blocks,
+    size_t num_byte_A,
+    size_t num_byte_B,
+    int64_t& Mc_blocks,
+    int64_t& Kc_blocks) {
+  Mc_blocks = Mt_blocks;
+  int64_t Nc_blocks = 1; // Nc_blocks is always 1
+  Kc_blocks = Kt_blocks;
+
+  // TODO: support multi-thread
+  if (num_threads != 1) {
+      return;
+  }
```

Comment:
Better to remove it in favor of https://github.com/pytorch/pytorch/pull/131024

Reply:
Removed